test/integration/create_classes_test.go

```diff
@@ -222,7 +224,9 @@ func TestCreateAllClasses(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if err := (&controller.Controllers{}).SetupWithManager(i); err != nil {
+	zl := zap.New(zap.UseDevMode(true))
+	log := logging.NewLogrLogger(zl.WithName("stack-gcp"))
```

Comment:
would it be useful/helpful to give a unique logger name for each test (or test file)?

Reply:
It would have to be for each test file, which could be useful! The controller logs output is currently listed under the invocation of the test, so we can associate that way, but naming them might be easier in the future if we want to consume them through any sort of ingestion engine.

Reply:
Updated per your suggestion :)